app-android/src/main/java/io/github/droidkaigi/confsched2022/impl/AndroidShareManager.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2022.impl
+
+import android.content.Context
+import androidx.core.app.ShareCompat
+import io.github.droidkaigi.confsched2022.ui.ShareManager
+
+class AndroidShareManager(private val context: Context) : ShareManager {
+    override fun share(text: String) {
+        ShareCompat.IntentBuilder(context)
+            .setText(text)
+            .setType("text/plain")
+            .startChooser()
```

Comment:
I don't think there is one, but may I ask you to add handling when `startChooser` throws exception as well?

Reply:
Added! 🚀